tardis/plasma/properties/nlte.py

```diff
@@ -14,4 +21,44 @@ class PreviousElectronDensities(PreviousIterationProperty):
     outputs = ('previous_electron_densities',)
 
 class PreviousBetaSobolevs(PreviousIterationProperty):
-    outputs = ('previous_beta_sobolevs',)
\ No newline at end of file
+    outputs = ('previous_beta_sobolevs',)
+
+class HeliumNLTE(ProcessingPlasmaProperty):
+    outputs = ('helium_population',)
+
+    def calculate(self, level_boltzmann_factor, electron_densities,
+        ionization_data, beta_rad, g, g_electron, w, t_rad, t_electrons,
+        delta, zeta_data, number_density):
+        helium_population = deepcopy(level_boltzmann_factor.ix[2])
+        # He I excited states
+        he_one_population = level_boltzmann_factor.ix[2].ix[0] * \
```

Comment:
@wkerzendorf Let me know if you think what I have now for the level_populations is better. I had another look and all of the level population calculations in the `HeliumNLTE` property are different to the level population calculations in `LevelNumberDensity` because they are all calculated relative to ground state populations rather than using the ion populations. So being able to call the same calculations would not be helpful. Let me know if that makes sense to you. I think it's tidier now anyway.